### PR TITLE
First Pass NCE Done Button Update 

### PR DESCRIPTION
**Pull request by StetsenTech**
**Ticket(s)**:
https://app.asana.com/0/1142638035116665/1154794977027306/f

**Summary:**
This is a first blanket pass through for getting the done button functionality to work properly. For the most part, this just replaces the identifiers for the section and question bodies so that the current javascript still functions with the new structure

There are still a number of things to do to get the done button functionality to 100%
- There are structural changes that need to be addressed in the code so that styling is applied properly (ex. answer section is now under the same parent as the question section)
- Some parts of sections are no longer direct siblings
- Answer/Correctness is no longer on the parent of inputs, so code needs to apply classes to the parent based on the child (unfortunately, CSS natively does not have browser support for this although it exists...)

### CODE REVIEW

**Comment by sadleb** [comment]
looks straightforward

File: braven_custom.js [hunk 1]
```diff
@@ -421,15 +421,15 @@ var bzNewUiHandlers = {
 
   // Show feedback on sort-to-match questions:
   '.for-match' : function(){
-    jQuery(this).parents('.bz-box').find('.sort-to-match').addClass('show-answers');
+    jQuery(this).parents('.content-section').find('.sort-to-match').addClass('show-answers');
   },
 
     // Generate self-evaluation results list:
   '.for-eval' : function(){
```
Comment (rshipp) [inline braven_custom.js]: not a blocker but do you know what this for-eval is? i haven't come across it in the html yet

Comment (StetsenTech) [reply braven_custom.js]: No idea. Been looking for it.

File: braven_custom.js [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ function collectStuffAfterBox(button) {
 
     <div class="show-content">
       <div wrappers>
-        <div class="bz-box">
+        <div class="content-section">
```
Comment (StetsenTech) [inline braven_custom.js]: Note to self: Update this docstrig

**Pull request merged**